src/settings/general-settings.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+
+import { Pane } from '@folio/stripes/components';
+
+export default class GeneralSettings extends React.Component {
+  static propTypes = {
+    label: PropTypes.object.isRequired,
```

Comment:
If it is React node expected there, I think a better prop type should be like PropTypes.node. Wondering whether it would work.

Reply:
Fixed